src/styling-panel-property-definition.js

```diff
@@ -0,0 +1,264 @@
+const createStylingPanelDefinition = () => {
```

Comment:
This is static at this point so does not need to be a function, same for the old definition

Reply:
Removed functions from both old and new definition.